src/jobs/models/jobsManager.ts

```diff
@@ -120,7 +147,10 @@ export class JobsManager {
         msg: `Updating catalog record ${mergedData.id as string} with merged metadata`,
       });
 
-      await this.publishCompletedUpdateRecord(job, task, mergedData, isSwap);
+      const layerName = this.getMapLayerName(previousLayerData);
+      await this.publishCompletedUpdateRecord(layerName, job, task, mergedData, isSwap);
+
+      await this.generateSeedJob(job, mergedData, previousLayerData, layerName, isSwap);
```

Comment:
Please move this method to be called  before publishCompletedUpdateRecord method - try catch  - if failed don't fail the process

Reply:
open issue - refactor the publishCompletedUpdateRecord - so the seed job will be created before publishing

Reply:
https://mapcolonies.atlassian.net/browse/MAPCO-4079

Reply:
added try-catch section + test